docs/open-source-quick-start/quick-start-installation-guide-on-gcp/query-your-postgres-data/index.md

```diff
@@ -14,7 +14,6 @@ If you were using the `default` or `secure` example scripts unedited in the last
 To query the good data in atomic.events, you will first you need to connect to your Postgres database.
 
 - Connect to the database using the username and password you provided when creating the pipeline, along with the `db_address` and `db_port` you noted down after the pipeline was created.
-    - If you need to reset your username or password you can [follow these steps](/docs/try-snowplow/try-snowplow-faq/index.md#my-database-login-details-are-not-working)
```

Comment:
This was not correct as it was pointing to AWS docs. I couldn’t find a good link for this so just removing.